src/store/plugins/veeValidate.js

```diff
@@ -124,6 +126,9 @@ export default (store) => {
     ),
     { params: ['isToken'] });
 
+  defineRule('token_to_an_btc_address',
```

Comment:
Why not just `address_btc`? The validation checks if the address string is a valid BTC address.